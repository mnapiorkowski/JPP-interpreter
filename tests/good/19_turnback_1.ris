void foo():
    let int i;
    while (i < 10):
        i++;
        println i;
    endwhile;
    i = 5;
    turnback;

string risotto(ref string s):
    s = s + "o";
    s = s + "t";
    turnback "Ris" + s;

int add_twice(int a, int b):
    a = a + b;
    turnback a;

void main():
    foo();

    let string s;
    println risotto(s);
    println s;

    println add_twice(4, 2);

    turnback;