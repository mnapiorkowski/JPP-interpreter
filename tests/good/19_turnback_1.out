1
2
3
4
5
6
7
8
9
10
6
7
8
9
10
Risotto
otto
8
8
otto
Risottootto
1
2
3
4
5
6
7
8
9
10
6
7
8
9
10
